docs/source/esp-idf_component.rst

```diff
@@ -1,20 +1,30 @@
-##############################
-Arduino as a ESP-IDF component
-##############################
+###############################
+Arduino as an ESP-IDF component
+###############################
+
+This method is recommended for advanced users. To use this method, you will need to have the ESP-IDF toolchain installed.
+
+For a simplified method, see `Installing using Boards Manager <https://docs.espressif.com/projects/arduino-esp32/en/latest/installing.html#installing-using-boards-manager>`_.
 
 ESP32 Arduino lib-builder
 -------------------------
 
-For a simplified method, see `lib-builder <https://github.com/espressif/esp32-arduino-lib-builder>`_.
+If you don't need any modifications in the default Arduino ESP32 core, we recommend you to install using the Boards Manager.
+
+Arduino lib builder will allow you to build your own version of Arduino core for Arduino IDE.
+
+To build your own Arduino core see `Arduino lib builder <https://github.com/espressif/esp32-arduino-lib-builder>`_
```

Comment:
It does not build the Arduino core. It builds ESP-IDF and collects all necessary headers, libraries and bootloaders. Basically everything inside `tools/sdk`.

Reply:
It is used when you need to have some specific ESP-IDF option that differs from what is already compiled in Arduino.

Reply:
What should the line be?

Reply:
Arduino Lib Builder is the tool that integrates ESP-IDF into Arduino. It allows you to customize the default settings used by Espressif and try them in Arduino IDE.